.evergreen/config.yml

```diff
@@ -1080,6 +1073,7 @@ buildvariants:
     has_packages: true
     packager_distro: rhel67
     packager_arch: s390x
+    compile_env: CMAKE_EXE=/opt/cmake/bin/cmake
```

Comment:
Why the special-case for rhel67-zseries-test? Can this be omitted so that `${compile_env|}` above can be made unnecessary?

Reply:
CMake does not ship a CMake binary for s390x, so we would need to build one. But: building a CMake would require a newer version of C++ and CMake than ships on RHEL 6.7, so rhel67 is just the unfortunate odd-one-out in this case. (Hopefully soon to be dropped as well)